build/project-template/__PROJECT_NAME__.xcodeproj/project.pbxproj

```diff
@@ -346,6 +362,10 @@
 				ENABLE_BITCODE = NO;
 				GCC_PRECOMPILE_PREFIX_HEADER = YES;
 				GCC_PREFIX_HEADER = "__PROJECT_NAME__/__PROJECT_NAME__-Prefix.pch";
+				HEADER_SEARCH_PATHS = (
```

Comment:
We'd better leave `"$(SRCROOT)/internal" here out of compatibility reasons. "$(SRCROOT)/internal/TKLiveSync/include"` can be left deleted.